Sasquatch/Sasquatch/Info.plist

```diff
@@ -16,14 +16,12 @@
 	<string>APPL</string>
```

Comment:
Or maybe the other way around, if this one can be shared as it's using project constants then probably the others could be removed.